src/check_python_versions/sources/poetry_pyproject.py

```diff
@@ -175,7 +195,7 @@ def _update_pyproject_toml_python_requires(
         return []
 
     table['tool']['poetry']['dependencies']['python'] = new_value
-    return pytomlpp.dumps(table).split('\n')
+    return dumps(table).split('\n')
```

Comment:
Note to self: investigate what tomlkit.dumps() returns and whether this should be using .splitlines() instead.

Note to self: make sure the unit tests verify that modifying a .toml file doesn't add or remove any trailing newlines at the end of the FILE.